journal_of_open_source_paper.md

```diff
@@ -0,0 +1,111 @@
+
+Deep learning thrives on the existence of vast and diverse datasets. 
+Collecting those in the real world is often either too complicated or expensive. 
+With `BlenderProc2`, we present a tool enabling the generation of vast and diverse datasets with a few lines of Python code.
+A particular focus is placed on the acknowledgment of the simulation-to-real gap and how to tackle this particular challenge in the dataset generation process.
+Even though the first version of `BlenderProc` was one of the first tools to generate photo-realistic, synthetic datasets, many more exist nowadays, compared in \autoref{tab:features} [@NDDS; @Stillleben; @nvisii; @habitat].
+In contrast to the first version of BlenderProc, BlenderProc2 relies on an easy-to-use python API, whereas the first version relied on a Yaml-based configuration approach [@denninger2019blenderproc; @denninger2020blenderproc].
+
+![An apple randomly sampled in the 3D FRONT dataset [@Front3D].](images/front_3d_object_sampling.jpg){ width=50% }
+
+\def\checkmarktikz{\tikz\fill[scale=0.4](0,.35) -- (.25,0) -- (1,.7) -- (.25,.15) -- cycle;} 
+\newcommand{\yesSign}{{\color{green}\checkmarktikz}}
+\newcommand{\noSign}{{\color{red}$\otimes$}}
+\newcommand{\journalCite}[2]{\protect\hyperlink{ref-#1}{#2}}
+
+\begin{table}[h]
+\centering
+\begin{tabular}{l|c | c | c | c | c}
+ & NDDS & NViSII & Habitat & Stillleben & \textbf{Ours} \\
+\hline
+semantic segm. & \yesSign & \yesSign & \yesSign & \yesSign & \yesSign \\
+depth rendering & \yesSign & \yesSign & \yesSign & \yesSign & \yesSign \\
+optical flow & \noSign & \noSign & \yesSign & \noSign & \yesSign \\
+surface normals & \noSign & \noSign & \yesSign & \yesSign & \yesSign \\
+object pose & \yesSign & \noSign & \yesSign & \yesSign & \yesSign \\
+bounding box & \yesSign & \noSign & \yesSign & \noSign & \yesSign \\
+\hline
+physics module & \yesSign & \noSign & \yesSign & \yesSign & \yesSign \\
+camera sampling & \yesSign & \noSign & \yesSign & \yesSign & \yesSign \\
+\hline
+based on an open-source renderer & \yesSign & \noSign & \noSign & \yesSign & \yesSign \\
+real-time & \yesSign & \noSign & \yesSign & \yesSign & \noSign \\
+\end{tabular}
+\caption{Main features present or not present in different simulators}
+\label{tab:features}
+\end{table}
+
+
+There are two groups of data generators.
+The first group is focused on speed and can generate several dozen images per second. 
+However, they generally rely on game engines to produce their images. These game engines focus on producing an image that can trick the human mind into believing a scene is real, which is not the same as generating a real image. 
+In contrast to that in the second group, the generation speed is not the focus but the final image's realism. 
+This realism is achieved by using a path tracer that traces the path of each light beam from a light source to the camera.
+Physical material properties then determine how the light interacts with the 3D scene and appears in the image.
```

Comment:
```suggestion
There are two groups of visual data generators.
The first group is focused on speed and can generate dozens of images per second by typically relying on game engines to produce their images. These game engines, however, focus on producing an image that can trick the human mind into believing that a scene is real, which is not the same as generating a real image. 
In contrast to that, in the second group we focus on the realism of the final images instead of on their generation speed. 
This realism is achieved by using a path tracer that follows the path of light beams from a light source to the camera.
Physical material properties then determine how the light in turn interacts with the 3D scene and appears in the image.
```